client/src/challenge/challenge-evaluations.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { Component, PropTypes } from 'react';
+import FlatButton from 'material-ui/FlatButton';
+
+import './challenge-evaluations.scss';
+import { api } from '../utils';
+import ApiWrapper from '../shared-components/api-wrapper';
+import UserComments from './user-comments';
+import UserComparison from './user-comparison';
+
+const endPoint = '/results/evaluate';
+const postEndPoint = '/results/evaluate';
+
+class ChallengeEvaluations extends Component {
+
+  static propTypes() {
+    return {
+      results: PropTypes.arrayOf(PropTypes.shape({
+        firstName: PropTypes.string.isRequired,
+        firstNameNumber: PropTypes.string.isRequired,
+        id: PropTypes.number.isRequired,
+        performanceDate: PropTypes.string.isRequired,
+        secondName: PropTypes.string,
+        secondNameNumber: PropTypes.string,
+        spotId: PropTypes.string.isRequired
+      })).isRequired
+    };
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      evaluating: true,
+      results: props.results
+    };
+    this.handleEvaluationToggle = this.handleEvaluationToggle.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleEvaluationToggle() {
+    const { evaluating, results } = this.state;
+
+    if (evaluating) {
+      this.setState({
+        evaluating: !evaluating,
+        results: results.map(({ id, ...rest }) => ({
+          ...rest,
+          id,
+          firstComments: this.refs[id].value().firstComments,
```

Comment:
I want this to be an instance variable of the class. So the call would look like `this.refs[id].value.firstComments`. Then in the class
```js
class UserComments extends Component {
...
get value() {
   return this.state;
}
...
}
```